extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/quantiles/DoublesSketchToHistogramPostAggregator.java

```diff
@@ -87,6 +110,12 @@ public PostAggregator getField()
     return splitPoints;
   }
 
+  @JsonProperty
+  public Integer getNumBins()
```

Comment:
Suggest doing `@JsonInclude(JsonInclude.Include.NON_NULL)` so the serialized form is cleaner when numBins is null.

(It would make sense to add this to getSplitPoints too.)